test/Client/CurlTest.php

```diff
@@ -365,7 +365,7 @@ public function testGetCurlHandle()
      */
     public function testHeadRequest()
     {
-        $this->client->setUri($this->baseuri . 'testRawPostData.php');
+        $this->client->setUri($this->baseuri . 'testHeadMethod.php');
```

Comment:
I'd expect a new test to be added here, instead of changing a pre-existing one (risky)